src/Filters/response.jl

```diff
@@ -128,3 +154,13 @@ end
 function hz_to_radians_per_second(hz, fs)
     hz * ((2 * pi) / fs)
 end
+
+function _is_sym(x::AbstractArray)
+    n = length(x) ÷ 2
+    return all([x[1+i] == x[end-i] for i in 0:n])
```

Comment:
```suggestion
    return all(x[1+i] == x[end-i] for i in 0:n-1)
```
Instantiating the vector is unnecessary, it creates an avoidable temporary and needs to perform all `==` tests, even if the first one already gives `false`. Also we can stop at `n-1`, I think.